packages/account-v2/src/modules/Onfido/OnfidoContainer.tsx

```diff
@@ -98,16 +97,17 @@ export const OnfidoContainer = ({
     }
 
     return (
-        <div className={qtMerge('flex flex-col items-center gap-800 p-800 lg:p-50')}>
+        <div className={twMerge('flex flex-col items-center gap-16 p-16 lg:p-0')}>
             <div
-                className={clsx(
-                    '[transition:transform_0.35s_ease,_opacity_0.35s_linear]origin-top opacity-1300 p-800',
-                    { '[display:none]': transitionEnd, 'scale-y-0 opacity-50': isOnfidoEnabled }
+                className={twMerge(
+                    '[transition:transform_0.35s_ease,_opacity_0.35s_linear]origin-top opacity-24 p-16',
+                    transitionEnd && 'd-none',
+                    isOnfidoEnabled && 'scale-y-0 opacity-50'
                 )}
             >
-                {/* TODO: Dummy div here replace with PoiConfirmWithExample */}
+                {/* Do this: Dummy div here replace with PoiConfirmWithExample */}
```

Comment:
TODO adds another sonarcloud issue, therefore should restrain from putting it

Reply:
I think its OK. Coz at least due to Soar cloud we'll have someone nudging to clear our TODOs 😸

Reply:
hahaha sure thing man, will leave the rest